web/template/payments.gohtml

```diff
@@ -68,6 +68,11 @@
             <a class="govuk-button govuk-button--secondary" href="{{ prefix (printf "/add-payment?id=%d" .Case.ID) }}">
                 Add payment
             </a>
+            {{ if .IsReducedFeesUser }}
+                <a class="govuk-button govuk-button--secondary" href="{{ prefix (printf "/apply-fee-reduction?id=%d" .Case.ID) }}">
+                    Apply fee reduction
+                </a>
+            {{ end }}
```

Comment:
Not sure how to test this in cypress.. doesn't seem like I can control what user I am and the role(s) that user would have